authentication/authentication-common/src/main/java/io/mosip/authentication/common/service/filter/BaseAuthFilter.java

```diff
@@ -95,7 +95,7 @@ protected void decipherAndValidateRequest(ResettableStreamHttpServletRequest req
 //			mosipLogger.debug(IdAuthCommonConstants.SESSION_ID, EVENT_FILTER, BASE_AUTH_FILTER, "Input Request: \n" + requestAsString);
 			requestWrapper.replaceData(requestAsString.getBytes());
 		} catch (IOException e) {
-			mosipLogger.error(IdAuthCommonConstants.SESSION_ID, EVENT_FILTER, BASE_AUTH_FILTER, e.getMessage());
+			mosipLogger.error(IdAuthCommonConstants.SESSION_ID, EVENT_FILTER, BASE_AUTH_FILTER, StringUtils.substring(e.getMessage(), 0, 500));

```

Comment:
e.getMessage() is just the error message, it need not be limited .. pls log the error as below
mosipLogger.error("<any message>",e);
So that any point of failure we will have the stacktrace to know the call hierarchy that caused the exception.

Reply:
@ase-101 no overloading method availabe in mosipLogger.error() to pass the exception. I will correct it pass the stack trace as string.